libs/shared/otel/src/lib/tests/node-tracing.integration.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { initTracing, reset, getCurrent } from '../node-tracing';
```

Comment:
Add mozilla license header. I'll mention it hear, but double check all new files.